src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_validators.py

```diff
@@ -22,3 +25,46 @@ def validate_deployment_name(namespace):
             namespace.deployment_name = os.path.splitext(template_filename)[0]
         else:
             namespace.deployment_name = 'deployment1'
+
+
+def _validate_lock_parameters(resource_group_name, resource_provider_namespace,
```

Comment:
It is preferable to give this method another name because its current name is too close to the `validate_lock_parameters`. A `_` as a prefix is not clear enough to distinguish two methods.